css/styles.css

```diff
@@ -0,0 +1,84 @@
+/************************************/
+/* HERO */
+/************************************/
+.hero {
+  /* display: flex; */
+  /* align-items: flex-end; */
+  /* justify-content: flex-start; */
+  /* background-image: url("https://lh3.googleusercontent.com/oJHsEQdk91V69_ePXPuU68R1o0Qu-1zqTxYBKU41n1o-37PDj3PhL2zdIL4Mh_Bp1ZqRDLllCYxdim-yZE-U3PMmCQbUgFKcJ_QImsH3qdDomYLGECoMwPSEKaqzJ8yhghq9_QMnRw=w2400");
+  background-size: 50%;
+  background-repeat: no-repeat;
+  background-position: center; */
+}
```

Comment:
just make sure to delete the unnecessary code or use this code for your final project :)